public/lesswrong-editor/src/ckeditor.js

```diff
@@ -107,6 +108,7 @@ const postEditorPlugins = [
 	ImageToolbar,
 	EasyImage,
 	ImageUpload,
+	CloudServicesPlugin,
```

Comment:
You needed to make a similar change to the comment editor's plugins list, since this plugin list is specific to the post editor. Failing to do this broke the comment editor (crash on load), and the breakage reached production.